news/2022-08-25-changes-to-osu-taiko-star-rating-part-1.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: post
+title: "Changes to osu!taiko star rating Part 1"
+date: 2022-08-25 12:00:00 +0000
+---
+
+It's no secret that osu!taiko star rating is overdue for a rework. Thanks to your responses in the community survey, we are now ready to unveil a series of planned updates and improvements to how difficulty is evaluated in the osu!taiko gamemode!
+
+![](/wiki/shared/news/2022-08-25-changes-to-osu-taiko-star-rating-part-1/star-rating.jpg)
+
+This effort was led by the newly formed [osu!taiko pp committee] (https://osu.ppy.sh/wiki/en/People/Performance_Points_Committee), in response to the results of the [osu!taiko SR and PP community survey](https://osu.ppy.sh/home/news/2022-02-07-taiko-mania-survey-results) held earlier this year. We are responsible for reviewing all changes regarding star rating or performance points in the osu!taiko gamemode. 
+
+## Roadmap for future
+For the full detailed roadmap, please click [here](https://docs.google.com/document/d/1Z5GC4DMqOVzeIERMSK3qpQaqjq-sVnhbuoxAwy9qxDs/edit#). This is a working document, and will periodically be updated with the latest developments.
+
+Difficulty in osu!taiko has historically been determined by 3 main components — Rhythm, Colour, and Stamina. Previous efforts have aimed to tackle all three at once, which has made it difficult to collect meaningful and targetted feedback. As such, we have opted for a modular approach, tackling new changes and reported issues one stage at a time towards more accurate solutions.
```

Comment:
The ending of the last sentence here kind of does not follow (*tackling (...) towards*?) Proposal:

```suggestion
Difficulty in osu!taiko has historically been determined by 3 main components — Rhythm, Colour, and Stamina. Previous efforts have aimed to tackle all three at once, which has made it difficult to collect meaningful and targetted feedback. As such, we have opted for a modular approach, tackling new changes and reported issues one stage at a time, in order to develop more accurate solutions.
```

Reply:
Applied in b824956